apps/minifront/src/components/tx-details/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { Card } from '@penumbra-zone/ui/components/ui/card';
 import { FadeTransition } from '@penumbra-zone/ui/components/ui/fade-transition';
-import { TxViewer } from './hash-parser';
```

Comment:
Not strictly part of this PR, but it was always bothering me how that file was named 😆